csen/download-models.sh

```diff
@@ -13,6 +13,12 @@ do
 		wget -nc http://data.statmt.org/bergamot/models/8bit-students/$lang/$model/lex.s2t.gz
 		wget -nc http://data.statmt.org/bergamot/models/8bit-students/$lang/$model/lex.s2t.bin
 		wget -nc http://data.statmt.org/bergamot/models/8bit-students/$lang/$model/vocab.spm
+        # rename files for consistency
+		mv model.8bit-finetuned.npz model.npz
```

Comment:
Hmmm we should really be fixing this on the server deployment side rather than the download shell script side.

Reply:
Done. Renamed on the server-side.